conbench/entities/history.py

```diff
@@ -454,12 +458,106 @@ def _query_and_calculate_distribution_stats(
     stats_df = history_df.sort_values("timestamp", ascending=False).drop_duplicates(
         ["case_id", "context_id"]
     )
+
     return {
         (row.case_id, row.context_id): (row.rolling_mean, row.rolling_stddev)
         for row in stats_df.itertuples()
     }
 
 
+def execute_history_query_get_dataframe(statement) -> Tuple[pd.DataFrame, Dict]:
+    """
+    Emit prepared query statement to database.
```

Comment:
@austin3dickey I wanted to have the raw BenchmarkResult objects after the history query, and here is what I ended up doing.

I do not really think that loading "too many" columns here is a problem. But we can always look into e.g. s.Numeric.